services/install-piwigo.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+function install {
+  # create service directory
+  mkdir -p /srv/piwigo/config
+
+  # create yml(s)
+  cat << EOF > /srv/piwigo/piwigo.yml
+version: "3.7"
+services:
+  piwigo:
+    image: linuxserver/piwigo
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Europe/London
+    volumes:
+      - /srv/piwigo/config:/config
+    ports:
+      - 9093:80
+    restart: unless-stopped
+    depends_on:
+      - db
+  db:
+    image: jsurf/rpi-mariadb
+    restart: unless-stopped
+    ports:
+      - 3306:3306
+    environment:
+      - MYSQL_ROOT_PASSWORD=\${MYSQL_ROOT_PASSWORD_VAR}
+      - MYSQL_DATABASE=\${MYSQL_DATABASE_VAR} 
+    volumes:
+      - /srv/piwigo/datadir:/var/lib/mysql
+EOF
+
+  # create .env with default values
+  cat << EOF  > /srv/piwigo/.env
+MYSQL_ROOT_PASSWORD_VAR=my-secret-pw
+MYSQL_DATABASE_VAR=piwigo
+EOF
+
+
+  # add autorun
+  cat << EOF > /srv/piwigo/autorun
+piwigo_autorun=true
+
+if [ "$piwigo_autorun" = true ]; then
+  treehouses services piwigo up
+fi
+EOF
```

Comment:
You could add two extra newlines between `fi` and `EOF`.
Not functionally important, but helps preserve the formatting in `/boot/autorun`